outputs/client.go

```diff
@@ -50,17 +52,23 @@ var ErrClientCreation = errors.New("Client creation Error")
 // EnabledOutputs list all enabled outputs
 var EnabledOutputs []string
 
+// files names are static fo the shake of helm and single docker compatibility
+const MutualTLSClientCertFilename = "/client.crt"
+const MutualTLSClientKeyFilename = "/client.key"
+const MutualTLSCacertFilename = "/ca.crt"
```

Comment:
Not all people use `Falcosidekick` in docker, I don't understand why these file names have to be static. Even with `helm`, we can adapt the values.

Reply:
I know these file names can be adapted, but I don't see any advantage in doing so. What I do see is 3 more lines in config.yaml and 3 more in the helm values.yaml file, which may be omitted.
Deploying with helm would be something like this:
`helm install --set config.mutualtlssupport.enabled=true --set config.mutualtlsfilespath=/client-certs/ --set-file config.mutualtlssupport.clientcert=./falcoskclient.crt --set-file config.mutualtlssupport.clientkey=./falcoskclient.key --set-file config.mutualtlssupport.cacert=./myca.crt falcosidekick `
But within falcosidekick container those files would be named client.crt, client.key and ca.crt.
I'm thinking as a user or even as an operator, and I don't know why should I care about the names of these files within the container. 
But if there's any reason this is important, I'll be glad making it configurable.

Reply:
IMHO:

* `config.mutualtlssupport.enabled=true` is useless, because `config.mutualtlsfilespath` is sufficient
* if we allow users which use helm to choose their key and certs, we should do the same for all users, some don't use helm and even docker.

Reply:
I'll get rid of the boolean flag, no problem.
Regarding 2nd issue: You can always choose the name of your files (helm or docker). In that helm example the files mounted in the falcosidekick container will be named client.crt, client.key and ca.crt as well. The set-file will only copy the content of the files and paste it on the secrets.yml template.

Reply:
boolean flag removed